pkg/kubectl/resource_printer_test.go

```diff
@@ -1474,6 +1488,12 @@ func TestPrintDeployment(t *testing.T) {
 			t.Fatalf("Expected: %s, got: %s", test.expect, buf.String())
 		}
 		buf.Reset()
+		// print deployment with '-o wide' option
+		printDeployment(&test.deployment, buf, PrintOptions{false, false, false, true, true, false, false, "", []string{}})
```

Comment:
Suggestion on `PrintOptions`:

1. we don't need `ShowAll` to be set to true (IIUC, `--show-all` only works on pods)
2. use tagged literals instead

Basically just change it to:
```go
PrintOptions{
  Wide: true
}
```

Bonus point if you can update this everywhere in this file